### ISSUE TITLE
Add check for Trust Authentication

### ISSUE THREAD

**Issue by AnnaPerry** [osisoft/PI-Security-Audit-Tools#2]
Security Best Practice #3: Use Windows authentication instead of PI Trust authentication

http://techsupport.osisoft.com/Troubleshooting/KB/KB00833

**Comment by FlorisAtMagion** [comment]
Check if Trusts are disabled. Now it gives a 'Severe' warning on disabled Trusts.

**Comment by AnnaPerry** [comment]
Hi @FlorisAtMagion, what is the exact warning you're getting? Currently, the check for trust authentication isn't enabled and shouldn't be generating any warnings. Perhaps a different Check is triggering?

**Comment by FlorisAtMagion** [comment]
AU20008 - MYPInode - piadmin is not used - Fail - Severe - Trust(s) that present weaknesses: AFtestHost; AFtestIP
Both Trusts are disabled.

**Comment by hpaul-osi** [comment]
Thanks for the clarification.  I've added a check of the flags attribute to the piconfig script used to avoid alerting on disabled trusts in the future.  This should be included in the next set of changes.

**Comment by FlorisAtMagion** [comment]
@hpaul-osi I just downloaded a new version and ran it against my environment, but under AU20008 it still comes up with a 'severe' warning for the disabled Trusts on piadmin.

**Comment by hpaul-osi** [comment]
Thanks for the comment, @FlorisAtMagion.  Did you run the _release_ (2.0.0.1 posted on July 25) or download the zip of the latest scripts (https://github.com/osisoft/PI-Security-Audit-Tools/archive/master.zip)?  

I ask because I cannot reproduce this issue on my test servers with zip of the latest code (disabled trusts are not shown and enabled trusts are).  Furthermore, in the latest code, the check for trusts using piadmin was moved to AU20002 with the piadmin usage for mappings.  AU20008 in the latest is the SPN check.  

If your reference to AU20008 above was a typo and you are in fact using the latest code and seeing this behavior with AU20002, then please send me a direct message on PI Square and we can discuss the details of the output or configuration you're looking at without worrying about disclosing any details of your system.  

Both the piconfig and powershell method were updated to only return enabled trusts with piadmin, so I don't see how this would still return those trusts.

@mode list
@table pitrust
@ostr pitrust
@sele piuser=piadmin,**flags=0**
@ends
@exit

$noncompliantTrusts = Get-PITrust -Connection $global:PIDataArchiveConnection `
									| Where-Object {$_.Identity -EQ 'piadmin' -and **$_.IsEnabled**} `
									| ForEach-Object {$_.Name}

**Comment by FlorisAtMagion** [comment]
[https://yox7hg-ch3302.files.1drv.com/y3mTBnduKfZ1VzaXOhRdYrAJ_GlJg-Cit6p0JNdaTYbXVCX5LCtzdspN4KLtUj9NuhoYrl2ynS_tlNLPATunVfaFvCyl9NgH_SK47PPGfBYe7gfGqMb7uWu604PPixHHL1PWAICxT2RrTMvSxs4FGjGcsfhkyv7rPT2OFrYNsvSJbs?width=1024&height=663&cropmode=none](url)

**Comment by hpaul-osi** [comment]
Thank you for posting the screenshot, but this looks like it could be using the 2.0.0.1 version, rather than the latest changes.  I recently posted the 2.0.1.0 version release which should include the fix for this issue.  Can you please try that one?  (https://github.com/osisoft/PI-Security-Audit-Tools/releases/download/v2.0.1.0/PI-Security-Audit-Tools-v2.0.1.0.zip)

**Comment by FlorisAtMagion** [comment]
Ah crystal!.
I should have downloaded it from GitHub of course, but I downloaded it from : 
https://techsupport.osisoft.com/Downloads/File/15af9717-b91f-47c6-bd4a-60a3d4084602
I tried to find how I got to this link. I think I know. If you search the techsupport site(search box) for the 'pi security audit', it is the first link, which has the download icon.
(http://search.osisoft.com/Pages/AllTechSupport.aspx?k=pi security audit)
